google/cloud/storage/examples/run_examples_utils.sh

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
+#
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+if [ -z "${PROJECT_ROOT+x}" ]; then
+  readonly PROJECT_ROOT="$(cd "$(dirname $0)/../../../.."; pwd)"
+fi
+source "${PROJECT_ROOT}/ci/colors.sh"
+
+################################################
+# Run all Bucket examples.
+# Globals:
+#   COLOR_*: colorize output messages, defined in colors.sh
+# Arguments:
+#   bucket_name: the name of the bucket to run the examples against.
+# Returns:
+#   None
+################################################
+run_all_bucket_examples() {
+  local bucket_name=$1
+  shift
+
+  for example in get-metadata; do
```

Comment:
I thought `get-metadata` might have been a function or something at first; can you store this in an array named as something that indicates what these strings mean, e.g. `BUCKET_SAMPLES_COMMANDS`?